packages/angular/cli/src/commands/mcp/tools/serve.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import { ChildProcess, spawn } from 'child_process';
+import { z } from 'zod';
+import { McpToolContext, declareTool } from './tool-registry';
+
+let devServerProcess: ChildProcess | null = null;
```

Comment:
Suggestion: I think we want to track this as a `Map<string /* project name */, ChildProcess>` so we can work with multiple projects at the same time.